src/controller/AutoCommissioner.cpp

```diff
@@ -287,6 +295,100 @@ CHIP_ERROR AutoCommissioner::NOCChainGenerated(ByteSpan noc, ByteSpan icac, Byte
     return CHIP_NO_ERROR;
 }
 
+const char * AutoCommissioner::StageToString(CommissioningStage stage)
```

Comment:
Would it make sense to move this into the CommissioningDelegate header where these are defined?

Reply:
Moved the declaration there, and the definition to a separate CPP file.